formats/ome-converter-plugin/tests/test_main.py

```diff
@@ -0,0 +1,109 @@
+"""Testing of Ome Converter."""
+import os
+import pathlib
+import sys
+from multiprocessing import cpu_count
+
+import filepattern as fp
+import numpy as np
+import requests
+from bfio import BioReader, BioWriter
+from PIL import Image
+from skimage import io
+
+from polus.plugins.formats.ome_converter.image_converter import image_converter
+
+dirpath = os.path.abspath(os.path.join(__file__, "../.."))
+sys.path.append(dirpath)
+
+inpDir = pathlib.Path(dirpath, "data/input")
+outDir = pathlib.Path(dirpath, "data/out")
+omeDir = pathlib.Path(dirpath, "data/outome")
+zarrDir = pathlib.Path(dirpath, "data/outzarr")
+if not inpDir.exists():
+    inpDir.mkdir(parents=True, exist_ok=True)
+if not outDir.exists():
+    outDir.mkdir(exist_ok=True, parents=True)
+if not omeDir.exists():
+    omeDir.mkdir(exist_ok=True, parents=True)
+if not zarrDir.exists():
+    zarrDir.mkdir(parents=True, exist_ok=True)
+
+imagelist = {
+    "0.tif": "https://osf.io/j6aer/download",
+    "img_r001_c001.ome.tif": "https://github.com/usnistgov/WIPP/raw/master/data/PyramidBuilding/inputCollection/img_r001_c001.ome.tif",
+    "00001_01.ome.tiff": "https://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/MitoCheck/00001_01.ome.tiff",
+}
+pattern = ".*"
```

Comment:
We should make these `fixtures` where appropriate.

Reply:
I tried to used pytest fixtures